tests/unit_tests/udf_wrapper_params/translation/single_model_single_batch_complete.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import PurePosixPath
+from exasol_udf_mock_python.connection import Connection
+from tests.unit_tests.udf_wrapper_params.translation.mock_translation import \
+    MockTranslationModel, MockTranslationFactory, MockPipeline
+
+
+def udf_wrapper():
+    from exasol_udf_mock_python.udf_context import UDFContext
+    from exasol_transformers_extension.udfs.models.translation_udf import \
+        TranslationUDF
+    from tests.unit_tests.udf_wrapper_params.translation. \
+        mock_translation_tokenizer import MockSequenceTokenizer
+    from tests.unit_tests.udf_wrapper_params.translation. \
+        single_model_single_batch_complete import \
+        SingleModelSingleBatchComplete as params
+
+    udf = TranslationUDF(
+        exa,
+        batch_size=params.batch_size,
+        pipeline=params.mock_pipeline,
+        base_model=params.mock_factory,
+        tokenizer=MockSequenceTokenizer)
+
+    def run(ctx: UDFContext):
+        udf.run(ctx)
+
+
+class SingleModelSingleBatchComplete:
+    """
+    single model, single batch, batch complete
+    """
+    batch_size = 2
+    data_size = 1
```

Comment:
I think, this is wrong, SingleModel, SingleBatch, Complete should be 

```suggestion
    batch_size = 2
    data_size = 2
```